thunder/tests/test_ops.py

```diff
@@ -557,3 +557,15 @@ def fn(a):
     jfn = thunder.jit(fn)
     a = torch.randn(5, 5, requires_grad=True)  # trigger grad transform
     assert_close(fn(a), jfn(a))
+
+
+@instantiate()
+def test_full_tensor_value(executor, device, dtype):
```

Comment:
Is there a reason for this to be a separate test and not expanding sample inputs for full OpInfo (`full_sample_generator`)?